framework/aster-frame/src/config.rs

```diff
@@ -19,6 +19,10 @@ pub const PAGE_SIZE_BITS: usize = 0xc;
 pub const KVA_START: usize = (usize::MAX) << PAGE_SIZE_BITS;
 
 pub const DEFAULT_LOG_LEVEL: Level = Level::Error;
+/// Whether to enable the debug log from scheduling.
+/// Only works when `DEFAULT_LOG_LEVEL` is `Debug`.
+/// Disabled by default.
+pub const SCHED_DEBUG_LOG: bool = false;
```

Comment:
Context switch after involving time slice and prority-based preemption becomes much more frequent than the current scheduler's, which results in an overwhelming number of sched-related debug logs in the output.

The config option, `SCHED_DEBUG_LOG`, is provided for those who prefer to dismiss logs related with the interface of the global scheduler, so that they won't be notified when a task is added/fetched/purged into/from the scheduler by default.

Reply:
Ok, for future expansion, I suppose adding a macro `sched_debug` for this task. The macro can be used the same way as `debug` macro. 
```rust
#[macro_export]
macro_rules! sched_debug {
    () => {};
    ($fmt: literal $(, $($arg: tt)+)?) => {
        if $crate::config::SCEHD_DEBUG_LOG {
            log::debug!($fmt $(, $($arg)+)?);
        }
    }
}
```

Reply:
Thx for the useful macro. It's added in `aster-frame::task::scheduler`.